packages/marshal/src/helpers/passStyle-helpers.js

```diff
@@ -2,8 +2,8 @@
 
 /// <reference types="ses"/>
 
-import '../types.js';
-import './internal-types.js';
+/** @typedef {import('../types').Checker} Checker */
```

Comment:
If we're sticking with JSDoc, maybe we can submit a PR for TypeScript to have a nicer syntax for this.

In regular TS this is clean:
```
import {type Checker} from '../types`
```

In JSDoc TS maybe,
```
/**
 * @typeimport '../types' {Checker}
 */
```